troposphere/static/js/components/login/OpenstackLoginForm.jsx

```diff
@@ -130,43 +131,11 @@ export default React.createClass({
             projectNameClasses = groupClasses,
             errorMessage = this.state.error_message != null ? "Login Failed: "+ this.state.error_message : null;
         let { provider, providerList} = this.state;
+        var renderLoginOrLoadingFunc = this.renderLoginOrLoading();
 
-        //FIXME: Shamefully using modal-footer : Get css-help later
         return (
-            <form>
-                <div className={usernameClasses}>
-                    <label htmlFor="username">
-                        Username
-                    </label>
-                    <input required
-                        type="name"
-                        className="form-control"
-                        id="username"
-                        value={this.state.username}
-                        onChange={this.onUsernameChange}
-                        onKeyPress={this.onEnterPressed}
-                        />
-                </div>
-                <div className={passwordClasses}>
-                    <label htmlFor="password">
-                        Password
-                    </label>
-                    <input required
-                        type="password"
-                        className="form-control"
-                        id="password"
-                        value={this.state.password}
-                        onChange={this.onPasswordChange}
-                        onKeyPress={this.onEnterPressed}
-                        />
-                </div>
-                <div className="login-screen-footer modal-footer">
-                    <span className="help-block">{errorMessage}</span>
-                    {this.renderLoginOrLoading()}
-                </div>
-            </form>
+            <LoginForm {...{username: this.state.username, password: this.state.password, usernameClasses, passwordClasses, errorMessage, renderLoginOrLoadingFunc, onUsernameChange: this.onUsernameChange, onPasswordChange: this.onPasswordChange, onEnterPressed: this.onEnterPressed}} />
```

Comment:
I will test it. I am not sure. I see an example in the code base is doing this in one line so that I assume it would work for one line. But, I could test it.